sample/src/main/java/me/saket/flick/sample/viewer/ImageViewerActivity.kt

```diff
@@ -98,39 +98,39 @@ class ImageViewerActivity : AppCompatActivity() {
   }
 
   private fun flickGestureListener(): FlickGestureListener {
-    return FlickGestureListener(this).apply {
-      onGestureInterceptor = object : OnGestureInterceptor {
-        override fun shouldIntercept(deltaY: Float): Boolean {
-          // Don't listen for flick gestures if the image can pan further.
-          val isScrollingUpwards = deltaY < 0
-          val directionInt = if (isScrollingUpwards) -1 else +1
-          return imageView.canScrollVertically(directionInt)
+    val contentHeightProvider = object : ContentSizeProvider {
+      override val heightForDismissAnimation: Int
+        get() = imageView.zoomedImageHeight.toInt()
```

Comment:
It feels weird to declare `heightForDismissAnimation` as a value when its value isn't fixed.